.husky/pre-push

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-yarn lint && yarn test && yarn check:dependencies && yarn check:packages
+yarn lint && yarn test && yarn check:deps && yarn check:registry
```

Comment:
each command can be on a separate line instead of being chained with `&&` but it doesn't really matter to me